### PR TITLE
probr - final fixes for passing in AWS + kube credentials.

### PR DESCRIPTION
**Pull request by thinkl33t**

### CODE REVIEW

**Comment by thinkl33t** [comment]
I've added a commit to this which sets the cluster name to be the same every time, so we know what the name of the generated kubeconfig will be.

I can't think of any reason this wouldn't be OK, but if we really to randomize it i should be able to do some additional fiddling to grab the terraform output for the kube config, dump it to a file of a known name, and pass it into probr.

File: .github/workflows/aws-eks-verify.yml [hunk 1]
```diff
@@ -74,8 +74,12 @@ jobs:
         id: probr
         uses: addnab/docker-run-action@v3
         with:
-          image: eknight/probr:v0.1.1 # https://github.com/probr/probr-docker
-          options: -v ${{ github.workspace }}/${{ env.PROBR_DIRECTORY }}:/probr/run
+          image: eknight/probr:v0.1.2-rc # https://github.com/probr/probr-docker
```
Comment (abdullahgarcia) [inline .github/workflows/aws-eks-verify.yml:77]: Is there a reason to use:

eknight/probr:v0.1.2-rc

?

**Review by abdullahgarcia — CHANGES_REQUESTED** [review]
Please address the question raised for clarity.

File: .github/workflows/aws-eks-verify.yml (hunk 1 above)
Comment (thinkl33t) [reply .github/workflows/aws-eks-verify.yml:77]: Thats the version eddie has been working on that adds the ability to pass in credentials on the commandline.

Comment (abdullahgarcia) [reply .github/workflows/aws-eks-verify.yml:77]: Let's move forward this way for now. We need a better/refined approach.

**Review by abdullahgarcia — APPROVED** [review]

**Pull request merged**